### PR TITLE
feat: add decision trees example

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
Adds the Colab Notebook corresponding to "[**Decision Trees: A Guide with Examples**](https://wandb.ai/sauravmaheshkar/Decision-Tree/reports/Decision-Trees-A-Guide-with-Examples--VmlldzoxMDE5Nzkw)"

### CODE REVIEW

File: colabs/scikit/wandb_decision_tree.ipynb [hunk 1]
```diff
@@ -0,0 +1,228 @@
+{
```
Comment (soumik12345) [inline colabs/scikit/wandb_decision_tree.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.    # Paste your api key here
</pre><p>Just <code>wandb.login()</code> would  suffice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (soumik12345) [inline colabs/scikit/wandb_decision_tree.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.    config.max_depth = 5
</pre><p>Configs should be initialized after <code>wandb.init()</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (SauravMaheshkar) [reply colabs/scikit/wandb_decision_tree.ipynb:1]: Addressed in [fc10336](https://github.com/wandb/examples/pull/492/commits/fc1033692793f2b7d684cedb5c1f0fc3e549b3c2)

Comment (SauravMaheshkar) [reply colabs/scikit/wandb_decision_tree.ipynb:1]: Addressed in [fc10336](https://github.com/wandb/examples/pull/492/commits/fc1033692793f2b7d684cedb5c1f0fc3e549b3c2)

**Review by soumik12345 — APPROVED** [review]

**Pull request merged**